src/components/shared/FormFileInput/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, InputWrapper } from './formFileInput.styles';
+
+/**
+ * Renders a form file input
+ *
+ * @param  {string}       acceptedType  Accepted files types.
+ * @param  {boolean}      multiple      Determines files quantity to single or many.
+ * @param  {Function}     setFiles      A function to set and update files.
+ * @param  {JSX.Element}  children      The child component as an interface for the file input.
+ *
+ * @return {JSX.Element}
+ */
+function FormFileInput({ acceptedTypes, multiple, onAddFiles, children }) {
+  const isFileTypeAccepted = (f) =>
+    !acceptedTypes || acceptedTypes.includes(f.type);
```

Comment:
```suggestion
  const isFileTypeAccepted = (file) =>
    !acceptedTypes || acceptedTypes.includes(file.type);
```

Also add docs for `file` props